### ISSUE TITLE
Error when running from command line

### ISSUE THREAD

**Issue by nscaife** [nbudin/agikit#1]
I installed this with `npm install -g @agikit/cli` as the readme states, but it throws immediately when I try to run `agikit` after installation. I'm on macOS 12.2.

```
node:internal/modules/cjs/loader:488
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/Types/Resources' is not defined by "exports" in /usr/local/lib/node_modules/@agikit/cli/node_modules/@agikit/core/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:703:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@agikit/cli/dist/Commands/build.js:9:21) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Node.js v17.4.0
```

**Comment by nbudin** [comment]
Hi @nscaife, thanks so much for the bug report!  I just tried this out on my MacBook, and it looks like this might be specific to Node.js 17 and above.  (I was able to successfully run the agikit command with Node 12 and 16 - didn't try the versions in between.)

I'm guessing Node 17 has imposed some additional requirements for explicitly defining exported subpaths in package.json.  I'll try to figure this out and release a patch ASAP.  In the meantime, if you want to just get agikit running, using any previous major version of Node should do the trick.  Thanks again, and sorry for the issue!

**Comment by nbudin** [comment]
It took a few tries, but I've just released version 0.7.2, which seems to work on Node 17.4.  Additionally, this release has a few other nice things, some of which had been sitting unreleased on the main branch for awhile:

* A new import entry point for the `@agikit/core` and `@agikit/react-editors` packages, which should make it unnecessary to do deep imports into these packages anymore
* Better diagnostic error messages for the LOGIC language server (used by the VSCode extension)
* A half-finished SOUND player, which currently only supports IBM PCjr SOUNDS (and doesn't yet correctly emulate the noise generator)
